pkg/cloudprovider/vsphere/loadbalancer/config/config_ini_legacy_test.go

```diff
@@ -17,35 +17,39 @@
 package config
 
 import (
-	"strings"
 	"testing"
 )
 
-func TestReadConfig(t *testing.T) {
+/*
+	TODO:
+	When the INI based cloud-config is deprecated. This file should be deleted.
+*/
+
+func TestReadINIConfig(t *testing.T) {
 	contents := `
 [LoadBalancer]
-ipPoolName = pool1
+ip-pool-name = pool1
```

Comment:
Why the field name changes here?

Reply:
We haven't cut a release since this functionality was introduced and I wanted to make the INI parsing values to be match other keys in all other configs. Currently, all keys use `-` between words. If we can't accept this change, I can revert but it would be nice to have uniform INI keys.

Reply:
@andrewsykim I think this is the only item left for this PR

Reply:
Talked with @andrewsykim and since we haven't had a release with the new LoadBalancer changes, there aren't any backward compatibility issues to be had and it's safe to update the INI keys to match all other key formats.